onnxruntime/python/onnxruntime_inference_collection.py

```diff
@@ -32,6 +32,52 @@ def get_ort_device_type(device_type: str, device_index) -> C.OrtDevice:
         raise Exception("Unsupported device type: " + device_type)
 
 
+class AdapterFormat:
```

Comment:
It would be harder to keep up to date, because it would not be used.

Reply:
sorry, I could have been more explicit. could we also use the generated python bindings instead of implementing our own python adapter format related APIs?